pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -1 +1,28 @@
-{ lib, callPackage }: lib.makeOverridable (args: (callPackage ./wrapper.nix { }) args)
+{ lib, callPackage }:
+lib.makeOverridable (
+  args:
+  let
+    buildFHSEnv = callPackage ./buildFHSEnv.nix { };
+    fhsenv = buildFHSEnv (
+      lib.attrsets.removeAttrs args [
+        "runScript"
+        "extraInstallCommands"
+        "meta"
+        "passthru"
+        "extraPreBwrapCmds"
+        "extraBwrapArgs"
+        "dieWithParent"
+        "unshareUser"
+        "unshareCgroup"
+        "unshareUts"
+        "unshareNet"
+        "unsharePid"
+        "unshareIpc"
+        "privateTmp"
+      ]
+    );
```

Comment:
Generally, I think I'd also like to get rid of this indirection somehow but that doesn't need to happen in this PR.

Reply:
Turns out this indirection is just plain unnecessary as only the `{,p}name` and `version` attrs of the `@args` pattern where used.